### DISCUSSION TITLE
A Year With The Framework Laptop 13

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#15] [Comments Archive]
# A Year With The Framework Laptop 13

My review of the Framework Laptop 13. I cover the assembly, setup, value, build, support, community, and much more.

https://vale.rocks/posts/a-year-with-the-framework-laptop-13

**Comment by anatom69** [comment]
Can you ship your framework laptop to me

**Reply by DeclanChidlow** [reply]
Only if you'll ship me one back. Preferably with an AMD board. ;)

**Reply by anatom69** [reply]
Can I ship you air instead
